### ISSUE TITLE
Docs have inconsistent TOCs

### ISSUE THREAD

**Issue by kincl** [openshift/openshift-docs#3170]
https://docs.openshift.org/latest/admin_guide/managing_projects.html

and 

https://docs.openshift.org/latest/admin_guide/managing_pods.html

If you notice, the TOC for Managing Pods is more complete.

**Comment by bfallonf** [comment]
@kincl Thanks for the feedback.

Can I ask what you think is missing from the pods TOC? They are very different objects, so it's makes sense that there's different content.

**Comment by kincl** [comment]
Hi @bfallonf,

Well it looks like it got cleared up, I was referring to the links on the left hand TOC for the entire docs site. From Managing Pods I was able to see more links to other articles like Assigning Unique External IPs for Ingress Traffic but under other pages like Managing Projects those links were not on the left hand side TOC.

All that being said, it appears to have been fixed today because every page displays the same left hand side TOC.

Thanks for looking into this, closing!
